eo-runtime/src/main/java/EOorg/EOeolang/EOrust.java

```diff
@@ -56,6 +58,10 @@
  * @checkstyle MethodNameCheck (100 lines)
  * @checkstyle LineLengthCheck (100 lines)
  * @checkstyle TypeNameCheck (5 lines)
+ * @todo #2283:90min Create Universe class. now its functionality is
```

Comment:
@levBagryansky Looks like a typo. Should word `now` start from a capital letter?